processors/src/SvtRawDataAnaProcessor.cxx

```diff
@@ -349,6 +408,7 @@ bool SvtRawDataAnaProcessor::process(IEvent* ievent) {
         }
         //std::cout<<count1<<std::endl;
         //std::cout<<count2<<std::endl;
+        std::cout<<"I got here 10"<<std::endl;
```

Comment:
go through and take out all these random couts you have for debugging, or at least add an if(debug) to them.

Reply:
I have taken out any that weren't commented and didn't mean something.